### DISCUSSION TITLE
What does the green circle on the line number mean?

### DISCUSSION THREAD

**Discussion by Bipinoli** [helix-editor/helix#9381] [Q&A] [answered]
I haven't put any breakpoint or anything there. It was picked up by default.
What does the green circle here mean?

![image](https://github.com/helix-editor/helix/assets/11765482/03374a21-d36c-42a3-a27d-0af32dff042f)

**Comment by kirawi** [comment]
It depends on your theme, but if it's not breakpoints then it's indicating there is some diagnostic on those lines.

**Comment by SublimeIbanez** [comment, marked as answer]
It's a diagnostic for both `e` and `event` on those lines (note the underlines).  Hover over them with the cursor and something should pop up at the top right to explain what's being notified.
